core/src/main/java/com/linecorp/armeria/internal/common/util/KeyStoreUtil.java

```diff
@@ -0,0 +1,186 @@
+/*
+ * Copyright 2023 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linecorp.armeria.internal.common.util;
+
+import java.io.BufferedInputStream;
+import java.io.EOFException;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.security.GeneralSecurityException;
+import java.security.KeyStore;
+import java.security.PrivateKey;
+import java.security.cert.Certificate;
+import java.security.cert.X509Certificate;
+import java.util.Arrays;
+import java.util.Enumeration;
+import java.util.List;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.io.ByteStreams;
+
+import com.linecorp.armeria.common.annotation.Nullable;
+
+import io.netty.util.internal.EmptyArrays;
+
+public final class KeyStoreUtil {
+    public static KeyPair load(File keyStoreFile,
+                               @Nullable String keyStorePassword,
+                               @Nullable String keyPassword,
+                               @Nullable String alias) throws IOException, GeneralSecurityException {
+        try (InputStream in = new FileInputStream(keyStoreFile)) {
```

Comment:
We could use `Files.newInputStream`
```suggestion
        try (InputStream in = Files.newInputStream(keyStoreFile.toPath())) {
```